cmd/query/app/http_handler_test.go

```diff
@@ -623,6 +633,52 @@ func TestGetOperationsLegacyStorageFailure(t *testing.T) {
 	require.Error(t, err)
 }
 
+func TestTransformOTLPSuccess(t *testing.T) {
+	reformat := func(in []byte) []byte {
+		obj := new(interface{})
+		require.NoError(t, json.Unmarshal(in, obj))
+		// format json similar to `jq .`
+		out, err := json.MarshalIndent(obj, "", "  ")
+		require.NoError(t, err)
+		return out
+	}
```

Comment:
you were write the first time about employing marshaling to ensure consistent formatting. I changed the expected fixture to be readable - also added a 2nd trace to test a branch in the translator